configs/default/FLWO-FLYWOOF411_5IN1_AIO.config

```diff
@@ -0,0 +1,120 @@
+# Betaflight / STM32F411 (S411) 4.2.8 Feb 15 2021 / 12:09:04 (101738d8e) MSP API: 1.43
+
+board_name FLYWOOF4115IN1AIO
```

Comment:
```suggestion
board_name FLYWOOF411_5IN1_AIO
```

Reply:
Thanks for the reminder, it has been edited